### DISCUSSION TITLE
Documents stuck in consume folder

### DISCUSSION THREAD

**Discussion by leginreklaw** [jonaswinkler/paperless-ng#525] [Support] [answered]
Hi everybody,

New to Paperless-ng but looks really great.
Have just completed a new install on my server, uploading documents via the web page works correctly as does uploading using the android app.
As I am remote from my server, and in a country that bans VPN's, I uploaded via SFTP some files (pdfs) to the consume folder, however these failed to process, with an error in the log that .part was an unknown file type. I guess the process was triggered before the file upload finished.

My question is, **is it possible to manually trigger the consume process to catch these files that now seem to be left behind in the consume folder?**

Apologies if this is documented somewhere, haven't been able to find it, only a note that inotify does not work properly on NFS file shares, which did not seem completely appropriate to my problem.

**Comment by jonaswinkler** [comment, marked as answer]
You could try `PAPERLESS_CONSUMER_POLLING=10` (example) nonetheless, this usually solves many issues with paperless not detecting files in the consume folder. File system notifications with `inotify` are only tested to work on local file systems, and I know that they play nice with samba shares. Other than that, no guarantees.

Looks like that FTP server stores uploaded files as "document.pdf.part" while uploading, and for some reason closes that file before it has been completely written. Maybe it creates that file to reserve the file name before opening it again for writing, and thus triggers the consumption process of paperless. This really is a quirk of the server software and something that's impossible to catch with file system notifications.

Also, paperless will try to pick up any files that are still in the consumption folder after a restart.

**Reply by leginreklaw** [reply]
Thank you - will give your suggestion a try.
